# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../grub/grub-0.97-AM_PROG_AS.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: am__fastdepCCAS does not appear in AM_CONDITIONAL
	     The usual way to define `am__fastdepCCAS' is to add AM_PROG_AS
	     http://trac.opensde.org/ticket/260

--- grub-0.97/configure.ac.orig	2007-01-16 23:44:06.000000000 +0100
+++ grub-0.97/configure.ac	2007-01-16 23:44:53.000000000 +0100
@@ -56,12 +56,7 @@
 
 AC_CHECK_TOOL(CC, gcc)
 AC_PROG_CC
-# We need this for older versions of Autoconf.
-_AM_DEPENDENCIES(CC)
-
-dnl Because recent automake complains about AS, set it here.
-CCAS="$CC"
-AC_SUBST(CCAS)
+AM_PROG_AS
 
 AC_ARG_WITH(binutils,
   [  --with-binutils=DIR     search the directory DIR to find binutils])
